### ISSUE TITLE
Template reference to ngModel produces error.

### ISSUE THREAD

**Issue by sgwatgit** [angular/vscode-ng-language-service#35]
Given the template

```html
<label for="name">Name</label>
<input type="text" class="form-control" id="name"
	   required
	   [(ngModel)]="model.name" name="name"
	   #name="ngModel" >
<div [hidden]="name.valid || name.pristine"
	 class="alert alert-danger">Name is required</div>
```

The ng language service will draw red squiggles under `name.valid` and `name.pristine` with the error

`[Angular] Identifier 'valid' is not defined. 'NgModel' does not contain such a member`

**Comment by chuckjaz** [comment]
Tracking in https://github.com/angular/angular/issues/15460

**Comment by chuckjaz** [comment]
Fixed in `0.1.1`

**Comment by javiercampos** [comment]
I have the same problem in `0.1.3`
